src/drivers/linux_pwm_out/linux_pwm_out.cpp

```diff
@@ -216,7 +216,11 @@ void task_main(int argc, char *argv[])
 		pwm_out = new NavioSysfsPWMOut(_device, _max_num_outputs);
 	}
 
-	if (pwm_out->init() != 0) {
+	//if using pca9685, _fd should large equal than 0 but in pca9685.h, init method return _fd>=0, it's alway be 1;
+	//So, I fix it
```

Comment:
Make comments more general and explicit. Do not use first person. Also for multi-line comments, use
```
/**
 *
 */
```
instead